### ISSUE TITLE
Covidcast CI R checks fail during "Set up R 3.5" step

### ISSUE THREAD

**Issue by JedGrabman** [cmu-delphi/covidcast#302]
The R CI checks are failing with several instances of the following warning:
```
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution 
by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see:
https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

This is affecting at least #297 and #275.

**Comment by chinandrew** [comment]
This is fixed in #298 and #290, merge in `main` or `r-pkg-devel` (depends on which your branched off) to your branch and it should be resolve.

**Comment by capnrefsmmat** [comment]
For #275 I'd recommending merging or rebasing on `r-pkg-devel`, since that's where we'll want to merge into (rather than main).
